docs/source/methods/neutron_physics.rst

```diff
@@ -1678,6 +1678,24 @@ default, the cutoff weight in OpenMC is :math:`w_c = 0.25` and the survival
 weight is :math:`w_s = 1.0`. These parameters vary from one Monte Carlo code to
 another.
 
+.. _survival_normalization:
+
+Survival Normalization
+----------------
+
+This parameter can be used to make the adjustable parameters of survival biasing,
+:math:`w_c` and :math:`w_s`, relative to the starting weight of a source particle.
+
+It is currently only implemented for phase-space sources (MCPL file sources
+and HDF5 surface sources). For these, the normalization of the parameters
+is done per source particle. That is, for each history, the parameters
+:math:`w_c` and :math:`w_s`, are multiplied by the start weight of the current
+history.
+
+This normalization is recommended for problems where numerous source particles are
+initialized below the survival biasing weight cutoff :math:`w_c`; to prevent them
+from being immediately rouletted. This can notably be the case for biased sources.
+
```

Comment:
```suggestion
```
We don't need a new section here, but in the existing survival biasing section it could be worth mentioning the the `weight_cutoff` and `weight_survive` used for russian roulette are the respective fractions of the initial particle weight.